sklearn/metrics/_classification.py

```diff
@@ -218,6 +230,16 @@ def accuracy_score(y_true, y_pred, *, normalize=True, sample_weight=None):
     0.5
     """
 
+    # Check y_true and y_pred is empty
+    len_y_true = _num_samples(y_true)
+    len_y_pred = _num_samples(y_pred)
+
+    if len_y_true == 0 and len_y_pred == 0:
```

Comment:
Working in my own PR, I was thinking that a try except clause is easier to read for later maintanance, because with defining a condition like `if len_y_true == 0 and len_y_pred == 0` we cannot easily see that this would cause a division by zero lateron.